apps/frontend/src/libs/modules/pwa/libs/types/types.ts

```diff
@@ -0,0 +1,9 @@
+type PWARegistrationConfig = {
+	immediate?: boolean;
```

Comment:
Boolean vars should start with `is`, `has`, `should` etc, something you can reply with true or false
```suggestion
	isImmediate?: boolean;
```